include/chimera/util.h

```diff
@@ -20,6 +20,9 @@ namespace chimera
 namespace util
 {
 
+constexpr char FLAG_NO_RENDER[]
+    = "CHIMERA_NO_RENDER_86c10773-c4e8-4b2f-95b3-137f8e4ea0da";
```

Comment:
Not sure if this is what you meant.

Reply:
I think so, there may be better ways to do this in recent C++ standards with `std::string` or `std::string_view`:
https://stackoverflow.com/a/43335753/2044807

Reply:
Thanks for the confirmation. Yeah, I was looking at the same post!